Sources/SPMBuildCore/BuiltTestProduct.swift

```diff
@@ -24,24 +24,39 @@ public struct BuiltTestProduct: Codable {
     public let packagePath: AbsolutePath
 
     /// The path of the test bundle.
+    ///
+    /// When the test product is not bundled (for instance, when using XCTest on
+    /// non-Darwin targets), this path is equal to ``binaryPath``.
     public var bundlePath: AbsolutePath {
-        // Go up the folder hierarchy until we find the .xctest bundle.
+        // Go up the folder hierarchy until we find the .xctest or
+        // .swift-testing bundle.
```

Comment:
formatting and wording nit

```suggestion
        // Go up the folder hierarchy until we find the `.xctest` or `.swift-testing` extension.
```

Reply:
Leaving out for now as I need to get this actually building, but will keep in mind.